src/adapters/adapter.types.ts

```diff
@@ -1,6 +1,13 @@
 import type { Stream } from 'stream';
+import { requestReplyAdapter } from './requestReply.adapter';
 
-export const ADAPTERS = ['default', 'foo'] as const;
+export const AdapterEnum = {
+  'request-reply': requestReplyAdapter,
+} as const;
+
+export const ADAPTERS = Object.keys(AdapterEnum);
```

Comment:
https://www.typescriptlang.org/docs/handbook/enums.html

To a potem kluczami w mapie (u Ciebie to jest AdapterEnum, ale nazwa jest błędna, bo tamto to jest mapa, a nie enum) mogą być wartości enuma

Reply:
zmienione